
[Growth chamber VR photosphere](https://photos.app.goo.gl/p4MdjcMeCVrbj23q6)

# Legume-rhizobia interactions in a complex microbiome

Ming-Dao Chia, Anna Simonsen, Andrew Almonte, Justin Borevitz

## Why?

- Rhizobia (nitrogen-fixing bacterial plant symbionts) are a vital source of bioavailable nitrogen, particularly in poor soils. Nutrients from legumes (rhizobia host plants) are important for agriculture and ecological regeneration.

- Many rhizobia species are well studied in single strain inoculation experiments, in culture, and in relation to their plant hosts.

- Less is known about their ecology in a complex microbiome, with closely related strains and uncultivated microbes.

- Recent developments in metagenomic methods allows studying abundance of rhizobia strains in nodules and free living environments with minimal culture or amplification bias.

## Aims

- Observing patterns of rhizobia strain abundance changes in legume nodules and associated soil.

- Observing patterns of rhizobia strain abundance changes in soil in the absence of a legume host.

- Identification of horizontal gene transfer events in known strains throughout the samples sequenced.

## Experiment Design

- Study species: legume *Acacia acuminata* with rhizobial symbiont *Bradyrhizobium japonicum* (4 strains).

- Plants are grown from surface sterilized seeds on autoclaved sand media for 8 weeks inside a growth chamber.

- A complex microbiome is inoculated from an external soil sample. Strains are inoculated at once as a balanced culture mix.

- DNA is extracted from sand and pots with a soil extraction kit.

- Long read sequencing (8-100 kb reads) is conducted with Oxford Nanopore MinION, allowing for strain-level identification and potential for observing horizontal gene transfer in single reads.

- Abundances of strains are measured through counting reads. 

- Potential horizontal gene transfer events are identified with bioinformatics.

## CurrentÂ Results

- Nanopore sequencing consistently yields some multigene metagenomic reads (>50 kb) with our samples.

- Combining long and short reads yields single contig reference genomes.

- Direct gDNA sequencing of sand and nodules produces reads that can be mapped to references.

## Future directions

- Link specific rhizobia genes or variants to abundance patterns

- Find potential co-inoculants to maintain stable free living rhizobia populations

- Create assays to identify stable inoculants for a variety of hosts and soil conditions

- Observe distribution between and within nodules of the same plant

- Correlate host genomic variance with abundance patterns

Contact: Ming-Dao.Chia@anu.edu.au
